File: C11/백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.c
#include <stdio.h>

int main(void){
	
	int pieces[6] = {1, 1, 2, 2, 2, 8};
	int input[6];
	
	for(int i = 0; i < 6; i++){
		scanf("%d", &input[i]);
	}
	
	for(int i = 0; i < 6; i++){
		int howMany = pieces[i] - input[i];
		printf("%d ", howMany);
	}
	
	return 0;
}

File: Java/백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int[] pieces = {1, 1, 2, 2, 2, 8};
        String[] input = br.readLine().split(" ");

        for (int i = 0; i < 6; i++) {
            int howMany = pieces[i] - Integer.parseInt(input[i]);
            System.out.print(howMany + " ");
        }
    }
}